dashboard/modules/serve/serve_rest_api_impl.py

```diff
@@ -296,4 +298,4 @@ async def run(self, server):
         def is_minimal_module():
             return False
 
-    return ServeRestApiImpl(dashboard_module_superclass)
```

Comment:
@shrekris-anyscale , can you review this change? This seems to be the incorrect behavior before. This function is supposed to return a class not an instance of the class.

Reply:
Yes, I think you're right. I'm not sure how we didn't catch this before, and how it didn't error before though.